package-lock.json

```diff
@@ -0,0 +1,8687 @@
+{
```

Comment:
Please delete this as we're using yarn instead of npm for this project, and having both a `yarn.lock` and a `package-lock.json` can cause some issues. Maybe run `yarn` as well to ensure that the extra dependencies you installed are inside of our yarn-lock as well!